packages/types/src/user/index.ts

```diff
@@ -17,6 +17,8 @@ export enum BackupEntryType {
   OAUTH = "OAUTH",
   LOCATION = "LOCATION",
   DELETE_OAUTH = "DELETE_OAUTH",
+  TOGGLE_SOCIAL_GRAPH = "TOGGLE_SOCIAL_GRAPH",
+  EDGE = "EDGE"
```

Comment:
The replace all didn't quite work, need to convert all `EDGE` sites to `TAP_GRAPH_EDGE`.